api/controllers/service_api/app/message.py

```diff
@@ -58,7 +58,9 @@ class MessageListApi(Resource):
         'conversation_id': fields.String,
         'inputs': fields.Raw,
         'query': fields.String,
+        'message_tokens': fields.Integer,
```

Comment:
The changes here are not in line with the topic of current PR, please create a new PR separately, thanks!